python/core/auto_generated/qgstemporalutils.sip.in

```diff
@@ -35,7 +35,9 @@ returns the maximal combined temporal extent of these layers.
     {
       QgsDateTimeRange animationRange;
 
-      QgsInterval frameDuration;
+      double frameTimeStep;
+
+      QgsUnitTypes::TemporalUnit frameTimeStepUnit;
```

Comment:
Can you revert this change too please? It's an api break, and we can safely use the same approach as we do elsewhere and rely on QgsInterval here.